_posts/2021-12-16-ngff.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: OME-NGFF 2021 updates
+tags: file-formats community
+categories: blog
+---
+
+In December 2020, we published two blog posts reporting on the development of
+next-generation file formats (NGFFs). We discussed public specifications for
```

Comment:
perhaps some like:
```
In November, we shared public examples of [multiscale images and labeled images in Novemebr] and [high-content screeening (HCS) datasets in December]
```
to make it clear (a) there are more images to be seen and (b) a date that can be associated. I guess if we had a bit more control of the layout, I could imagine some form of thumbnails that show each of the above, but we can always think about that after the fact.

Reply:
Guess the downside is that it then takes away from the thumbnails at the bottom.